sw/device/silicon_creator/rom/e2e/BUILD

```diff
@@ -2362,6 +2365,106 @@ test_suite(
     tests = ["epmp_init_otp_" + lc_state for lc_state, _ in get_lc_items()],
 )
 
+platform_target(
+    name = "rom_with_fake_keys_fpga_cw310_elf",
+    testonly = True,
+    platform = OPENTITAN_PLATFORM,
+    target = "//sw/device/silicon_creator/rom:rom_with_fake_keys_fpga_cw310.elf",
+)
+
+rust_binary(
```

Comment:
The convention of the repository so far has been that all test harnesses go into `sw/host/` while `sw/device/` only contains the device code and the bazel definition of the test.